rwengine/src/engine/GarageController.cpp

```diff
@@ -1,702 +0,0 @@
-#include "GarageController.hpp"
```

Comment:
For future changes: When renaming files manually git does not automatically keep track of previous history. If you had used `git mv GarageController.cpp Garage.cpp`, then git should have created a much much nicer history / diff.
That would also potentially have helped in review.

Fixing this retroactively is a bit harder, but theoretically possible (essentially manually changing it back, and then using `git mv`, then force push - do at your own risk)

Reply:
Nevermind, GitHub might just handle this poorly - in the "Big Refactor" commit this is displayed properly.